src/auth/internal/AuthIntegrationInterface.ts

```diff
@@ -115,13 +115,17 @@ function initializeAuthClientListeners() {
   }
 }
 
+/**
+ * This function fires when the idle timer goes off. It either extends the auth
+ * session or performs logout actions
+ */
 function idleTimerAction(hasAuthed: boolean, logoutActions: () => void) {
-  if (!isAuthStytch()) return;
-
-  if (authState.client.session.getSync()) {
+  if (isAuthStytch() && authState.client.session.getSync()) {
     authState.client.session.authenticate({
       session_duration_minutes: 1440
     });
+  } else if (global.firebase) {
+    // I don't think firebase lets us extend the session manually, without issuing a token from the BE
```

Comment:
maybe just reword the comment so it's more confident.

// Firebase doesn't let us extend the session manually,...